Vegeting/Screens/GroupList/ViewController/GroupListViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  GroupListViewController.swift
+//  Vegeting
+//
+//  Created by kelly on 2022/10/26.
+//
+
+import UIKit
+
+class GroupListViewController: ViewController {
+    var clubList = [Club]() {
+        didSet {
+            collectionView.reloadData()
```

Comment:
p3;
UI관련된 작업은 main queue에서 해주세요!

Reply:
콜렉션뷰에 나타날 클럽리스트 배열의 변경이 있으면 relaod해줘야 한다고 판단해 위와 같이 작성하였는데,
혹시 어떻게 작성하라는 건가요??

Reply:
콜렉션 뷰를 reload 함으로써 UI의 변경이 생기므로 해당 함수는 UI와 관련된 작업입니다. UI관련된 작업을 main queue에서 해야하는 이유는 UIKit이 thread-safe 하지 않기 때문입니다! 
```suggestion
Dispatchqueue.main.async { 
            collectionView.reloadData()
}
```
관련 [블로그](https://jeong9216.tistory.com/515)를 첨부합니다!

Reply:
오 그렇군요!! 배워갑니다~
반영완료 [8bacd29](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/56/commits/8bacd2918fb68d4df37b2b90b0f31bf9209a658b)